src/sidebar/components/Excerpt.tsx

```diff
@@ -163,7 +178,9 @@ function Excerpt({
           'inline-block w-full'
         )}
         data-testid="excerpt-content"
-        ref={contentElement}
+        ref={element => {
+          contentElement.current = element ?? undefined;
+        }}
```

Comment:
This is a bit cumbersome thing to do, which happens due to type definitions.

The `ref` prop expects a ref objects that handles `Element | null`, while `useRef` works with `Element | undefined`.

I've never been so sure why the type is defined like this in react/preact, as setting the ref object itself actually works at runtime.

One option is setting `ref={contentElement as any}` 🤷🏼‍♂️

Reply:
Initializing this ref to `null` eliminates need for these acrobatics. See comment above.

Reply:
Aha! It looks so obvious now 😄